packages/smart-wallet/src/smartWallet.js

```diff
@@ -315,10 +356,33 @@ export const prepareSmartWallet = (baggage, shared) => {
         .returns(M.promise()),
       publishCurrentState: M.call().returns(),
       watchPurse: M.call(M.eref(PurseShape)).returns(M.promise()),
+      repairUnwatchedSeats: M.call().returns(),
+      updateStatus: M.call(M.any()).returns(),
+      addContinuingOffer: M.call(
+        M.or(M.number(), M.string()),
+        AmountShape,
+        M.remotable('InvitationMaker'),
+        M.or(M.record(), M.undefined()),
+      ).returns(M.promise()),
+      purseForBrand: M.call(BrandShape).returns(M.promise()),
+      logWalletInfo: M.call().rest(M.arrayOf(M.any())).returns(),
+      logWalletError: M.call().rest(M.arrayOf(M.any())).returns(),
+      // XXX is there a tighter guard for a bigMapStore than M.any()?
```

Comment:
there isn't, but there could be a shape for it. consider,
```suggestion
      // TODO define a shape for bigMapStore 
```
even better, make an issue and mention it here

Reply:
`M.remotable()` seems like the right shape.
or with a label such as `M.remotable('mapStore')`.

https://github.com/Agoric/agoric-sdk/blob/5a06ded0d99e0e8ee3e78191ff4a538c091628ff/packages/swingset-liveslots/src/collectionManager.js#L843

Reply:
I assumed we could do better by including the actual properties (methods) of a mapStore. M.remotable() is incrementally better than M.any() but then the shape can't change later to one that is a local object with methods.

Reply:
Patterns only admit passables, such as a remotable or a copyRecord with properties (whose values are passable), but local objects with methods aren't passable, IIUC.

Reply:
`M.remotable('mapStore')` is indeed better than `M.any()`. Thanks

Reply:
Thanks @dckc and sorry for steering your wrong @Chris-Hibbert . I neglected to think through it, and was yet was hasty in my certainty. I plead guilty by reason of fever.